koku/reporting/rate/models.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Models for identity and access management."""
+
+from uuid import uuid4
+
+from django.db import models
+
+
+class Rate(models.Model):
+    """A rate for calculating costs.
+
+    A rate is (price * metric_usage / timeunit).
+    """
+
+    uuid = models.UUIDField(default=uuid4, editable=False,
```

Comment:
What is this uuid for? 

As it stands it's not the primary key, so there will also be an auto-incrementing id column here. Personally, I think we only need that auto-id field.

Reply:
We leverage uuid for most models so that we aren't tied to an internal db id. I'm guessing this is done for consistency.

https://github.com/project-koku/koku/blob/master/koku/api/provider/models.py#L31

Reply:
Yeah, I get that. I think it makes less sense inside the tenant schema. No other models there use a uuid right now. Still, not holding back on this comment, just was looking for an answer of what it was really getting us.

Reply:
I don't have a good reason for it. Mostly just following the pattern we're using in other models.